src/style/layers.css

```diff
@@ -1,92 +1,207 @@
+  left: auto;
+  top: 9px;
+  border: 0;
+  border-radius: 0;
 }
 
-.ngm-displayed-container.buttons {
-  justify-content: flex-end;
-  margin-bottom: 5px;
+ngm-layers-item .ui.dropdown .menu .item:hover {
+  background-color: #C5F6FA;
 }
 
-.ngm-displayed-container.transparency {
+/* KML upload */
+ngm-layers-upload button {
+  cursor: pointer;
   display: flex;
-  margin-left: 25px;
-  height: 30px;
+  align-items: center;
+  letter-spacing: 0.25px;
+  font-weight: bold;
+  color: #0B7285;
+  background-color: #F1F3F5;
+  border: 2px dashed #0B7285;
+  margin-top: 9px;
+  padding-left: 10px;
+  height: 46px;
+  width: 100%;
 }
 
-.ngmmidgrey {
-  background-color: #999 !important;
-  color: white !important;
+ngm-layers-upload .ngm-upload-icon {
+  pointer-events: none;
+  background-color: #0B7285;
+  margin-top: 0;
+  margin-left: 12px;
+  width: 20px;
+  height: 20px;
 }
 
-.ngmlightgrey {
-  background-color: #c7c7c7 !important;
+.ngm-extension-panel .ui.header {
+  margin-top: 7px;
 }
 
-.ngmverylightgrey {
-  background-color: #e9e9e9 !important;
+.ngm-extension-panel .ui.divider {
+  margin-top: 15px;
+  margin-bottom: 8px;
+  border-top: 1px solid #DFE2E6 !important;
+  border-bottom: 1px solid #DFE2E6 !important;
 }
 
-.ngm-displayed-container label.displayed {
-  color: red;
+/* Layer catalog */
+ngm-catalog .ui.accordion, ngm-catalog .ui.accordion .content {
+  margin: 0 !important;
+  padding: 0 !important;
 }
 
-ngm-layers-item {
+ngm-catalog .ui.accordion .title {
   display: flex;
-  flex-direction: column;
+  align-items: center;
 }
 
-ngm-layers .ui.divider {
-  margin: 5px -5px;
+ngm-catalog .ui.accordion .title label {
+  cursor: pointer;
+  margin-left: 5px;
+  color: #212529;
 }
 
-ngm-layers .ui.segment {
-  padding: 5px 5px;
+ngm-catalog .ui.accordion .title.first-level {
+  font-weight: bold;
 }
 
-ngm-layers-upload {
-  display: flex;
-  flex-direction: column;
-  margin-top: 10px;
-  margin-bottom: 10px;
+ngm-catalog .ui.accordion .title.active label, .ngm-catalog-entry .displayed {
+  color: #A83526 !important;
 }
 
-ngm-layers-upload ngm-layers {
-  margin-top: 10px;
+ngm-catalog .ui.accordion .title:hover label, .ngm-catalog-entry:hover label {
+  color: #FF0000 !important;
 }
 
+/* Dropdown icon */
+ngm-catalog .ui.accordion .ngm-dropdown-icon {
+  background-color: #0B7285;
+}
 
+ngm-catalog .ui.accordion .title.active .ngm-dropdown-icon {
+  background-color: #A83526;
+  transform: rotate(90deg);
+}
+
+ngm-catalog .ui.accordion .title:hover .ngm-dropdown-icon {
+  background-color: #FF0000;
+}
+
+ /* Catalog entry with custom checkbox */
+.ngm-catalog-entry {
+  /* display: block; */
+  position: relative;
+  padding-left: 28px;
+  margin-left: 3px;
+  margin-top: 8px;
+  margin-bottom: 8px;
+  /* align-items: center; */
+  cursor: pointer;
+}
+
+/* Hide the browser's default checkbox */
+.ngm-catalog-entry input {
+  position: absolute;
+  opacity: 0;
+  height: 0;
+  width: 0;
+}
+
+/* Custom checkbox */
```

Comment:
I also added a custom checkbox (https://github.com/swissgeol/ngm/blob/new_ui/src/style/ngm-checkbox.css), we should merge it somehow

Reply:
done in https://github.com/swissgeol/ngm/pull/653/commits/e284f40cf4a345fcc1eea44f833a3ba24d648f6f